tgui/packages/tgui/interfaces/ChemMaster.js

```diff
@@ -0,0 +1,457 @@
+            onClick={() => act("insert")}
+            bold>
+            Insert Beaker
+          </Button>
+        </Dimmer>
+      )}
+      <ReagentGraph container={container} />
+      <ReagentList container={container}
+        renderButtons={(reagent) => {
+          return (
+            <>
+              <Button
+                px={0.75}
+                mr={1.5}
+                icon="search"
+                color="green"
+                title="Analyze"
+                onClick={() => act("analyze", { reagent_id: reagent.id })}
+              />
+              <Button
+                px={0.75}
+                mr={1.5}
+                icon="filter"
+                color="red"
+                title="Isolate"
+                onClick={() => act("isolate", { reagent_id: reagent.id })}
+              />
+              <Button
+                px={0.75}
+                mr={1.5}
+                icon="times"
+                color="red"
+                title="Flush all"
+                onClick={() => act("flush", { reagent_id: reagent.id })}
+              />
+              <Button
+                px={0.75}
+                icon="minus"
+                color="yellow"
+                title="Flush variable amount of units"
+                bold
+                onClick={() => act("flushinput", { reagent_id: reagent.id, amount: remove_amount })}>
+                {remove_amount}u
+              </Button>
+            </>
+          );
+        }}
+      />
+    </SectionEx>
+  );
+};
+
+
+export const SelectIconModal = (props, context) => {
+  const { data } = useBackend(context);
+  const { pill_icons, bottle_icons, patch_icons } = data;
+
+  const { icon_list, set_icon } = props;
+
+  const button_image = (item, index) => (
+    <Button key={index}
+      onClick={() => { set_icon(index); }}>
+      <Image
+        height="32px" width="32px"
+        pixelated
+        src={`data:image/png;base64,${icon_list !== ICON_LIST_PILLS ? item[1] : item}`}
+        style={{
+          'vertical-align': 'middle',
+          'horizontal-align': 'middle',
+        }} />
+      {
+        icon_list !== ICON_LIST_PILLS && (
+          <Box
+            textAlign="center" bold>{item[0]}u
+          </Box>
+        )
+      }
+    </Button>
+  );
+
+  return (
+    <Modal full
+      width="80%" mx="auto"
+      backgroundColor="#1c2734"
+    >
+      {
+        icon_list === ICON_LIST_PILLS && (
+          <SectionEx capitalize
+            title="Select pill icon">
+            {
+              pill_icons.map(button_image)
+            }
+          </SectionEx>)
+      }
+      {
+        icon_list === ICON_LIST_BOTTLES && (
+          <SectionEx capitalize
+            title="Select bottle icon">
+            {
+              bottle_icons.map(button_image)
+            }
+          </SectionEx>)
+      }
+      {
+        icon_list === ICON_LIST_PATCHES && (
+          <SectionEx capitalize
+            title="Select patch icon">
```

Comment:
Use something other than the term "icon", like style or type. That was it maintains emersion preferably.